src/main/java/io/cdap/plugin/google/common/GoogleDriveClient.java

```diff
@@ -111,6 +111,6 @@ public void checkRootFolder() throws IOException {
   }
 
   public void isFolderAccessible(String folderId) throws IOException {
-    service.files().get(folderId).execute();
```

Comment:
why is this change needed?

Reply:
Not needed, added while debugging, forgot to remove. Will remove that change in future commit.